indico/modules/categories/schemas.py

```diff
@@ -6,15 +6,22 @@
 # LICENSE file for more details.
 
 from indico.core.marshmallow import fields, mm
+from indico.modules.categories import Category
 from indico.modules.categories.models.event_move_request import EventMoveRequest
 from indico.modules.events import Event
 from indico.modules.users import User
 
 
-class CategoryEventSchema(mm.SQLAlchemyAutoSchema):
+class EventSchema(mm.SQLAlchemyAutoSchema):
```

Comment:
Maybe `MoveRequestEventSchema` fits better since that's what we are using it for?

Reply:
Renamed :+1: